problem.py

```diff
@@ -1,46 +1,56 @@
 import os
+import numpy as np
 import pandas as pd
 # import rampwf as rw
 from sklearn.model_selection import StratifiedShuffleSplit
 
-import local_workflow
-import local_scores
+import workflow.local_workflow as local_workflow
+import workflow.scores as local_scores
+
 
 problem_title = 'Mars craters detection and classification'
 
-# _target_column_name = 'class'
-# _prediction_label_names = [0, 1, 2, 3]
-# # A type (class) which will be used to create wrapper objects for y_pred
-# Predictions = rw.prediction_types.make_multiclass(
-#     label_names=_prediction_label_names)
+# A type (class) which will be used to create wrapper objects for y_pred
+from workflow.predictions import Predictions
+
 # An object implementing the workflow
 workflow = local_workflow.ObjectDetector(
     test_batch_size=16,
-    chunk_size=256,
-    n_jobs=8,
-    # img_file_extension='png',
-    # n_classes=len(_prediction_label_names),
-)
+    chunk_size=50,
+    n_jobs=8)
 
 score_types = [
-    local_scores.Accuracy(name='acc'),
-    local_scores.NegativeLogLikelihood(name='nll'),
+    local_scores.Ospa(),
+#    local_scores.Accuracy(name='acc'),
+#     local_scores.NegativeLogLikelihood(name='nll'),
 ]
 
 
 def get_cv(folder_X, y):
-    _, X = folder_X
-    cv = StratifiedShuffleSplit(n_splits=1, test_size=0.2, random_state=57)
-    return cv.split(X, y)
+    #_, X = folder_X
+    #cv = StratifiedShuffleSplit(n_splits=1, test_size=0.2, random_state=57)
+    #return cv.split(X, y)
+
+    # for now dummy CV that has one fold with all data for both train/valid
+    return [(slice(None), slice(None)), ]
 
 
 def _read_data(path, f_name):
-    df = pd.read_csv(os.path.join(path, 'data', f_name))
-    X = df['id'].values
-    y = df['class'].values
-    folder = os.path.join(path, 'data', 'imgs')
-    return (folder, X), y
+    src = np.load('data/images_quad_77.npy', mmap_mode='r')
+    labels = pd.read_csv("data/quad77_labels.csv")
+
+    y = [list(labels[labels.id == '77_{0}'.format(i)][
+                  ['x_p', 'y_p', 'radius_p']].itertuples(name=None,
+                                                         index=False))
+         for i in range(src.shape[0])]
+
```

Comment:
It took me 30 sec to understand how the `y` is actually created 😄 
Maybe add a tiny bit of explanation for a non pandas-ninja..